models/stg_stripe.yml

```diff
@@ -101,7 +101,7 @@ models:
         tests:
           - not_null
       - name: amount
-        description: Amount intended to be collected by this payment. A positive integer representing how much to charge in the smallest currency unit (e.g., 100 cents to charge $1.00 or 100 to charge ¥100, a zero-decimal currency)
+        description: Amount intended to be collected by this payment. A positive integer representing how much to charge in the major currency unit (e.g., 1 equals a charge of $1.00 or a charge ¥100, a zero-decimal currency)
```

Comment:
These read a bit confusing when talking about the ¥ charge. Maybe we should add here that if you're currency does not contain decimal units then you should leverage the variable to turn off the division.

Reply:
I updated the language and pulled it into the docs.md file to avoid repetition, and added `"{{ doc('convert_values') }}"` at the end of every description for all amount-based fields. Let me know what you think